src/highlightjs/material-dark.css

```diff
@@ -0,0 +1,114 @@
+/** From https://github.com/Kelbster/highlightjs-material-dark-theme */
```

Comment:
Can we consume this from npm? Just copying like this would require the license to be included as well.